apply-notch-hax.sh

```diff
@@ -7,8 +7,24 @@ if [ $UID -eq 0 ]; then
 fi
 
 WORK="$HOME/.cache/unity8-notch-hax"
-DEVICE="$(getprop ro.product.device)" # e.g. 'yggdrasil'
+DEVICE="$1"
+
+if [ -z $DEVICE ]; then
+	echo "Attempting to detect device name"
+	DEVICE="$(getprop ro.product.device)" # e.g. 'yggdrasil'
+fi
+
+# getprop workaround for quirky devices
```

Comment:
This comment could be better as it's not that the devices are "quirky", it's just a different systemimage ([the halium-generic_arm64 one](https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-generic_arm64/) in this case to be more specific which afaik is also used on Erfan's UBports GSI).

Alternatively the comment could just be omitted and the info can be conveyed in the `echo` below as `Trying backup getprop` doesn't exactly tell much (and is just being verbose for the sake of it).